.github/workflows/linux_matrix_e2e_on_nightly.yml

```diff
@@ -22,33 +22,33 @@ jobs:
           - name: AMD/HIP
             runner: '["Linux", "amdgpu"]'
             image: ghcr.io/intel/llvm/sycl_ubuntu2204_nightly:build
-            extra_cmake_args: -DHIP_PLATFORM="AMD" -DAMD_ARCH="gfx1031"
             extra_image_opts: --device=/dev/kfd
+            extra_cmake_args: -DHIP_PLATFORM="AMD" -DAMD_ARCH="gfx1031"
             target_devices: ext_oneapi_hip:gpu
             reset_gpu: false
 
           - name: Intel
             runner: '["Linux", "gen9"]'
             image: ghcr.io/intel/llvm/sycl_ubuntu2204_nightly:latest
-            extra_cmake_args:
             extra_image_opts: -u 1001
+            extra_cmake_args:
             target_devices: ext_oneapi_level_zero:gpu;opencl:gpu;opencl:cpu
             reset_gpu: true
 
           - name: ESIMD Emu
             runner: '["Linux", "x86-cpu"]'
             image: ghcr.io/intel/llvm/sycl_ubuntu2204_nightly:latest
-            extra_cmake_args:
             extra_image_opts: -u 1001
+            extra_cmake_args:
             target_devices: ext_intel_esimd_emulator:gpu
             reset_gpu: false
-    uses: ./.github/workflows/linux_single_e2e_on_nightly.yml
+    uses: ./.github/workflows/linux_single_e2e.yml
     with:
       name: ${{ matrix.name }}
       runner: ${{ matrix. runner }}
       image: ${{ matrix.image }}
+      image_options: --device=/dev/dri --privileged --cap-add SYS_ADMIN ${{ matrix.extra_image_opts }}
       extra_cmake_args: ${{ matrix.extra_cmake_args }}
-      extra_image_opts: ${{ matrix.extra_image_opts }}
```

Comment:
are we using extra_image_opts someplace else?   I thought -u 1001 was required for some reason that I can't remember ( same user as when containers are created? )

Reply:
Resolved offline.